Add info in kubectl get triggerauthentication (#4668)
kedacore/keda: CHANGELOG.md

### Improvements

- **General**: Metrics Adapter: remove deprecated Prometheus Metrics and non-gRPC code ([#3930](https://github.com/kedacore/keda/issues/3930))

> - **General:**: Add ScaledObject/ScaledJob Names when kubectl get triggerauthentication/clustertriggerauthentication([#796](https://github.com/kedacore/keda/issues/796))

Reviewer: ```suggestion
- **General:**: Add ScaledObject/ScaledJob names to output of `kubectl get triggerauthentication/clustertriggerauthentication` ([#796](https://github.com/kedacore/keda/issues/796))
```
